che-theia-plugins.yaml

```diff
@@ -1,22 +1,30 @@
 version: 1.0.0
 plugins:
+
+  # github/vscode-pull-request-github/0.20.0
```

Comment:
Maybe it's better to skip version in comments like this. I don't see some profit to have it there, but each time when the contributor will update the plugin he should remember about this comments.

Reply:
no problems, I've just added it for easier managing the list of plugins